lxd/db.go

```diff
@@ -12,6 +12,11 @@ import (
 	_ "github.com/mattn/go-sqlite3"
 )
 
+var (
+	DbErrAlreadyDefined = fmt.Errorf("already exists")
+	NoSuchImageError    = errors.New("No such image")
```

Comment:
What's the difference between `fmt.Errorf` and `errors.New` here? Should we just use one or the other?